searchcore/src/tests/proton/documentdb/feedview/feedview_test.cpp

```diff
@@ -594,7 +594,8 @@ struct FixtureBase
     void putAndWait(const DocumentContext &docCtx) {
         FeedTokenContext token(_tracer);
         PutOperation op(docCtx.bid, docCtx.ts, docCtx.doc);
-        runInMaster([&] () { performPut(token.ft, op); });
+        runInMaster([this, ft=std::move(token.ft), &op] () { performPut(std::move(ft), op); });
```

Comment:
Consider adding mutable specifier to the lambda expression.

Reply:
Fixed